StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1513UnitTests.cs

```diff
@@ -1013,5 +1013,29 @@ public void TestMethod(string extraSupport)
 
             await VerifyCSharpDiagnosticAsync(testCode, DiagnosticResult.EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
         }
+
+        [Fact]
+        [WorkItem(3360, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3360")]
+        public async Task TestLineFeedEndOfLinesAsync()
+        {
+            var testCode =
+                "public class TestClass\n" +
+                "{\n" +
+                "}\n" +
+                "// Hello";
```

Comment:
💡 You can define a `ReplaceLineEndings` helper like this:
https://github.com/dotnet/roslyn/blob/f64507a8d6b6c3e01f8d1e05ce97552f3c37b341/src/Compilers/Test/Core/FX/StringExtensions.cs

This allows you to use verbatim strings like all the other tests, and just call the extension method to assign the line endings you want.

Reply:
Absolutely. Would you like to keep the "fancy" conversions, or just assume CRLF in the input?